stdlib/src/os/os.mojo

```diff
@@ -244,3 +244,53 @@ fn abort[
         print(message, flush=True)
 
     return abort[result]()
+
+
+# ===----------------------------------------------------------------------=== #
+# remove/unlink
+# ===----------------------------------------------------------------------=== #
+fn remove(path: String) raises:
+    """Removes the speficied file.
+
+    Args:
+      path: The path to the file.
+
+    """
+    var error = external_call["unlink", Int8](path._as_ptr())
```

Comment:
**Suggestion** It doesn't matter too much in practice, but I'd use `Int` instead of `Int8` here as it more closely matches the UNIX function return type as seen [here](https://man7.org/linux/man-pages/man2/unlink.2.html).  This would follow the element of least surprise to future readers even though the return values all technically fit in the type domain of an `Int8` as you have it now.

Reply:
This should be Int32 since it matches the C int type